src/js/components/DataSort/DataSort.js

```diff
@@ -0,0 +1,81 @@
+import React, { useContext, useMemo } from 'react';
```

Comment:
It looks like DataSort is not working as expected. Sorting doesn't change the table.

Uploading Dec-20-2022 10-43-33.mp4…

Reply:
fixed